x-pack/plugins/ingest_manager/public/applications/ingest_manager/sections/overview/index.tsx

```diff
@@ -46,26 +41,30 @@ export const IngestManagerOverview: React.FunctionComponent = () => {
       leftColumn={
         <EuiFlexGroup direction="column" gutterSize="m">
           <EuiFlexItem>
-            <EuiText>
-              <h1>
-                <FormattedMessage
-                  id="xpack.ingestManager.overviewPageTitle"
-                  defaultMessage="Ingest Manager"
-                />
-                <AlphaBadge
-                  label={i18n.translate('xpack.ingestManager.alphaBadge.labelText', {
-                    defaultMessage: 'Experimental',
-                  })}
-                  title={i18n.translate('xpack.ingestManager.alphaBadge.titleText', {
-                    defaultMessage: 'Experimental',
+            <EuiFlexGroup responsive={false} gutterSize="s" alignItems="center">
+              <EuiFlexItem grow={false}>
+                <EuiTitle size="l">
+                  <h1>
+                    <FormattedMessage
+                      id="xpack.ingestManager.overviewPageTitle"
+                      defaultMessage="Ingest Manager"
+                    />
+                  </h1>
+                </EuiTitle>
+              </EuiFlexItem>
+
+              <EuiFlexItem grow={false}>
+                <EuiBetaBadge
+                  label={i18n.translate('xpack.ingestManager.betaBadge.labelText', {
+                    defaultMessage: 'Beta',
                   })}
-                  tooltipContent={i18n.translate('xpack.ingestManager.alphaBadge.tooltipText', {
+                  tooltipContent={i18n.translate('xpack.ingestManager.betaBadge.tooltipText', {
                     defaultMessage:
-                      'This plugin might change or be removed in a future release and is not subject to the support SLA.',
+                      'This plugin is not recommended for use in production environments. Please help by reporting bugs and sending feedback.',
```

Comment:
You could shorten this and be more specific by saying : "Please report bugs in our Discuss forum." (If possible, make this an active link to the Discuss forum.)

Reply:
Not possible to make it a link because it's inside of a tooltip, but I like the addition so that it's more specific. Updated